### PR TITLE
add-adhoc-may-june-for-mentors

### PR DESCRIPTION
**Pull request by IdayatSanni**
## Description
Mentors that confirmed availability for May/June ad-hoc session were updated such as Hersi Kopani, Adegbenga Agoro  and Nada Kartouch 

## Change Type
- [x] Mentor Update

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [x] I have tested my changes locally.
- [x] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: _data/mentors.yml [hunk 1]
```diff
@@ -1830,7 +1830,7 @@
 
 - name: Hersi Kopani
   disabled: false
-  sort: 100
+  sort: 200
```
[the remark this answers is not in the document]
Comment (dricazenck) [reply _data/mentors.yml:1833]: Please also update the pull request descriptions and check the requirements. 
image

**Review by dricazenck — APPROVED** [review]

**Comment by dricazenck** [comment]
there is another mentor also to update: Shailaja.

File: _data/mentors.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (IdayatSanni) [reply _data/mentors.yml:1833]: Ok, I will do that now

**Review by khairahscorner — APPROVED** [review]

**Comment by khairahscorner** [comment]
I'll go ahead and merge this but as Adriana mentioned, it'd be nice to attach screenshots in the PR description showing that the mentors are now available for the specified months✅

**Pull request merged**